changelog.md

```diff
@@ -6,7 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.1.1](https://github.com/adrianprelipcean/express-rate-limit-postgresql/releases/tag/v1.1.1)
+## [1.2.0](https://github.com/adrianprelipcean/express-rate-limit-postgresql/releases/tag/v1.2.0)
+
+### Breaking
+
+- Bumped `express-rate-limit` to
+  [7.0.1](https://github.com/express-rate-limit/express-rate-limit/releases/tag/v7.0.1)
+- Bumped `sinon` to
+  [16.0.0](https://github.com/sinonjs/sinon/blob/main/CHANGES.md#1600)
+- Bumped `license-checker-rseidelsohn` to
+  [4.2.7](https://github.com/RSeidelsohn/license-checker-rseidelsohn/releases/tag/v4.2.7)
```

Comment:
I wouldn't even mention these in the changelog. 

I might call out the moving of express-rate-limit from `dependencies` to `peerDependencies`, though.